samples/tab-device-permissions/nodejs/src/styles/layouts.js

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { makeStyles } from "@fluentui/react-components";
+
+export const getFlexRowStyles = makeStyles({
+  root: {
+    display: "flex",
+    minHeight: "0px",
+  },
+  fill: {
+    width: "100%",
+    height: "100%",
+  },
+  smallGap: {
+    "> :not(:last-child)": {
+      marginRight: "0.5rem",
+    },
+  },
+  spaceBetween: {
+    justifyContent: "space-between",
+  },
+  hAlignStart: {
+    justifyContent: "start",
+  },
+  hAlignCenter: {
+    justifyContent: "center",
+  },
+  hAlignEnd: {
+    justifyContent: "end",
+  },
+  vAlignStart: {
+    alignItems: "start",
+  },
+  vAlignCenter: {
+    alignItems: "center",
+  },
+  vAlignEnd: {
+    alignItems: "end",
+  },
+});
+
+export const getFlexColumnStyles = makeStyles({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    minHeight: "0px",
+  },
+  fill: {
+    width: "100%",
+    height: "100%",
+  },
+  smallGap: {
+    "> :not(:last-child)": {
+      marginBottom: "0.5rem",
+    },
+  },
+  spaceBetween: {
+    justifyContent: "space-between",
+  },
+  hAlignStart: {
+    alignItems: "start",
+  },
+  hAlignCenter: {
+    alignItems: "center",
+  },
+  hAlignEnd: {
+    alignItems: "end",
+  },
+  vAlignStart: {
+    justifyContent: "start",
+  },
+  vAlignCenter: {
+    justifyContent: "center",
+  },
+  vAlignEnd: {
+    justifyContent: "end",
+  },
+  scroll: {
+    overflowY: "auto",
+    msOverflowStyle: "auto",
+    maxHeight: "100vh",
+  },
+});
+
+export const getFlexItemStyles = makeStyles({
+  grow: {
+    flexGrow: 1,
+  },
+  noShrink: {
+    flexShrink: 0,
+  },
```

Comment:
There are lots of warnings in your code, please check if we can fix those?
![image](https://user-images.githubusercontent.com/59052547/210078813-a7a868fe-aa33-4b92-8d9d-5364699c6b68.png)

Reply:
unable to understand about these